backend_new/prisma/seed-helpers/multiselect-question-factory.ts

```diff
@@ -24,7 +24,7 @@ export const multiselectQuestionFactory = (
     subText: `sub text for ${text}`,
     description: `description of ${text}`,
     links: [],
-    options: multiselectOptionFactory(optionalParams?.numberOfOptions || 0),
```

Comment:
if numberOfOptions is no longer used, can it be removed from the optionalParams?

Reply:
good catch, removed